### PR TITLE
Add Menu tests

### PR DESCRIPTION
**Pull request by ShivamPokhriyal**

### CODE REVIEW

**Comment by dimagimon** [comment]
[32% (0.0%) vs master 32%](https://jenkins.dimagi.com/job/commcare-android-pr-job-builder/461977/)

File: app/instrumentation-tests/src/org/commcare/androidTests/MenuTests.kt [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+        onView(withText(startsWith("Submit Forms")))
+                .check(matches(isDisplayed()))
+        Espresso.pressBack()
+        onView(withText(formDumpConfirmation))
+                .check(doesNotExist())
+        InstrumentationUtility.gotoHome()
+
+        openAdvancedOption(4)
+        Intents.intended(IntentMatchers.hasComponent(CommCareVerificationActivity::class.java.name))
+        onView(withId(R.id.home_gridview_buttons))
```
Comment (shubham1g5) [inline app/instrumentation-tests/src/org/commcare/androidTests/MenuTests.kt:161]: we can't check for toast messages as an indication of success ?

**Review by shubham1g5 — APPROVED** [review]

File: app/instrumentation-tests/src/org/commcare/androidTests/MenuTests.kt (hunk 1 above)
Comment (ShivamPokhriyal) [reply app/instrumentation-tests/src/org/commcare/androidTests/MenuTests.kt:161]: No, unfortunately we can't with espresso.

**Comment by dimagimon** [comment]
[32% (0.0%) vs master 32%](https://jenkins.dimagi.com/job/commcare-android-pr-job-builder/461978/)

**Pull request merged**